app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/ProtvistaPdbValidation.tsx

```diff
@@ -332,13 +359,48 @@ function useBar() {
 
         tooltip25
             .append("text")
-            .text("Quartile-25: " + quartile25 + " Å")
+            .text(`Quartile-25: ${Number(metrics["quartile 25"])}  Å`)
```

Comment:
That will shown NaN if values is undefined. Maybe skip the whole tooltip25 if the metric is not found? Same for median and q75.

Reply:
do you mean something like this? `${metrics["quartile 25"] ? Number(metrics["quartile 25"]) : ''}`

Reply:
I meant to completely skip the tooltip if we had nothing to show for it, but let's go with this and let Adrian decide. Only, it can be simplified: `.text(`Quartile-75: ${metrics["quartile 75"] ?? "-"}  Å`)`

Reply:
okay got it.